test/inductor/test_torchinductor.py

```diff
@@ -6296,6 +6296,23 @@ def fn(x):
                         - metrics.generated_cpp_vec_kernel_count
                     ) == 0
 
+        def test_redundant_to_node_elimination_bf16(self):
+            def fn(x, y):
+                res = x + y
```

Comment:
Can you explain how this test would produce redundant to_dtype to float?

Reply:
This test case invokes `mean` and the lowering of `mean` inserts `to_dtype`. https://github.com/pytorch/pytorch/blob/master/torch/_inductor/lowering.py#L3464